.travis.yml

```diff
@@ -12,6 +12,10 @@ dist: trusty
 # https://github.com/travis-ci/travis-ci/issues/9033
 sudo: required
 
+before_install:
+  - if [ "$TRAVIS_OS_NAME" = "osx" ]; then brew update          ; fi
+  - if [ "$TRAVIS_OS_NAME" = "osx" ]; then brew install cppcheck; fi
```

Comment:
Please add to `Brewfile` in the root folder, instead.

Reply:
This is already taken care.

Reply:
How is it taken care of? [Brewfile](https://github.com/libexpat/libexpat/blob/master/Brewfile) does not contain an entry for cppcheck yet, in this pull request.

Reply:
I will upload the change today,  once the cppcheck mode check is taken care by me.

Reply:
Please check new pull request https://github.com/libexpat/libexpat/pull/290